src/olympia/shelves/models.py

```diff
@@ -3,7 +3,7 @@
 from olympia.amo.models import ModelBase
 
 SHELF_TYPES = ('categories', 'collections', 'extension',
-               'recommendations', 'search', 'theme')
+               'recommended', 'search', 'theme')
```

Comment:
what's the difference between `extension`, `theme`, `search`, and `recommended` (and `categories`)?  It looks like they all do exactly the same thing?

Reply:
I initially thought they all went to separate endpoints, but now that it's really just two, it seems redundant to list out types. Can we rename it to endpoints and just keep `search` and `collections`?

Reply:
yeah I think that's best.